packages/frontend/src/pages/Spaces.tsx

```diff
@@ -39,6 +47,10 @@ const Spaces: FC = () => {
             <Stack spacing="xl" w={900}>
                 <PageHeader>
                     <PageBreadcrumbsWrapper>
+                        {/*test purposes only! will remove before merging*/}
+                        <PageBreadcrumbs items={breadCrumbsItems}>
+                            All spaces
+                        </PageBreadcrumbs>
```

Comment:
since this is already working can you replace breadcrumbs on all content management pages? I mean all spaces/spaces/dashboard and chart pages?

Reply:
yep! will do it in a seperate PR, just want to keep things small 👀